### PR TITLE
Translated string for adding word to dictionary and few others

### PR DESCRIPTION
**Pull request by krunose**
Translated string for adding unknown word to dictionary (new feature) and fix for few others.

### CODE REVIEW

**Comment by krunose** [comment]
Fixe tabs to spaces.

File: locale/retext_hr.ts [hunk 1]
```diff
@@ -144,6 +149,17 @@
...
+        <location filename="editor.py" line="493"/>
+        <source>%d w | %d a | %d c</source>
+        <comment>broj riječi, alfanumerički znakovi, svi znakovi</comment>
+        <translation>%d m | %d a | %d c</translation>
+    </message>
+    <message>
+        <location filename="editor.py" line="184"/>
+        <source>Add to dictionary</source>
+        <translation>Dodaj riječ u rječnik</translation>
+</message>
```
Comment (mitya57) [inline locale/retext_hr.ts]: If you can then please fix it also here, then I will merge it.

**Comment by krunose** [comment]
Did indentation of closing message tag

**Comment by mitya57** [comment]
Merged, thank you!

**Comment by mitya57** [comment]
@krunose “Could not parse file contents…” (tab.py:188) and “Cannot save to file which…” (window.py:860) messages have duplicate `<translation>` entries. Can you please fix that or tell me which ones I should delete?

**Pull request merged**